topics/howtosim.rst

```diff
@@ -0,0 +1,255 @@
+   cachefile            : XXXXX                # Name of EOF cache file (e.g. eof.cache.fileF)
+   centerfile           :                      # Read position and velocity center from this file
+   constheight          : true                 # Use constant disk scale height
+   dbods                : disk.bods            # Disk particle output file
+   deproject            : Exponential          # set deprojection model (Exponential or MN)
+   disk_mass            : XXXXX                # Mass of stellar disk (e.g. 0.0125)
+   dtype                : exponential          # Disk type for condition (one of: constant, gaussian, mn, exponential)
+   expcond              : true                 # Use analytic density function for computing EmpCylSL basis
+   gentype              : asymmetric           # DiskGenType string for velocity initialization (Jeans, Asymmetric, or Epicyclic)
+   halofile1            : XXXXX                # File with input halo model (your .model file here)
+   evolved              : false
+   hbods                : halo.bods            # Halo particle output file
+   ignore               : false                # Recompute EOF regardless of existence
+   multi                : false                # Use multimass halo
+   ndisk                : XXXXX                # Number of disk particles (e.g. 1000000)
+   ngas                 : 0                    # Number of gas particles
+   ngparam              : 3                    # Number of gas particle parameters
+   nhalo                : XXXXX                # Number of halo particles (e.g. 10000000)
+   report               : true                 # Report particle progress in EOF computation
+   runtag               : XXXXX                # Label prefix for diagnostic images (e.g. 'run1')
+   scale_height         : XXXXX                # Scale height for disk realization (e.g. 0.002)
+   scale_length         : XXXXX                # Scale length for disk realization (e.g. 0.01)
+   threads              : 1                    # Number of lightweight threads
+   zerovel              : true                 # zero center of mass and velocity
+   suffix               : ""
+
+Parameters that have values set to 'XXXXX' are those that users are encouraged to change. Example values are given in the
+comment following the parameter value. The remaining parameters are "expert parameters" and generally do not need to be changed
+unless you have a good reason to use a different value. Note that the ``exponential`` disk is
+exponential in R and :math:`sech^2` in z.
+
+To generate the initial conditions, you would run:
+
+.. code-block:: python
+
+   mpirun gendisk --config gendisk_step1.yml 
+
+in the terminal, replacing ``gendisk_step1.yml`` with the name of your YAML file that specifies the ``gendisk`` parameters. If
+you set ``ignore = false``, it may take a moment to generate the cylindrical bases cache file. The cylindrical bases will be
```

Comment:
*moment* is a bit of an understatement.  But I sort of like it.  Maybe *a few minutes* would be a better indicator.